docs/guides/update-guide/810-to-820.md

```diff
@@ -0,0 +1,21 @@
+---
+id: 810-to-820
+title: Update 8.1 to 8.2
+description: "Review which adjustments must be made to migrate from Camunda Platform 8.1.x to Camunda Platform 8.2.0."
+---
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from Camunda Platform 8.1.x to 8.2.0 for each component.
+
+## Server
+
+### Zeebe
+
+No changes are required to update from 8.1 to 8.2.
+
+The default [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) algorithm changed from **Vegas** to **additive-increase/multiplicative-decrease (AIMD)**.
+
+:::note
+The AIMD algorithm improved in 8.2. This may change the performance characteristics.
```

Comment:
Is it worth clarifying here how the algorithm improved here, and how this specifically may change performance characteristics?

Reply:
Thanks for your review. I have updated the note.

Reply:
Thank you!